src/LOAuth.ts

```diff
@@ -112,6 +112,42 @@ class LOAuth {
     return btoa(JSON.stringify(state));
   }
 
+  /**
+   * This method looks for a cookie set by PHC-Login that allows the
+   * applications to exchange auth tokens over different subdomains
+   * see phc-login/src/util/handleAuthResponse
+   */
+  private _getDomainCookieAuthState() {
+    try {
+      if (!document.cookie) {
+        return;
+      }
+      const cookie = document.cookie
+        .split('; ')
+        .find((cookie) => cookie.startsWith(`${AUTH_STORAGE_KEY}=`));
+
+      const value = cookie && cookie.split('=')[1];
+      if (!value) {
+        return;
+      }
+
+      const storedData = JSON.parse(value);
+      this.token = this.client.createToken({
```

Comment:
Not blocking, but I find the pattern of this method modifying `this.token` and then line 190 checking that something happened a little hard to follow.

If I was writing it, I would have returned the token and set it in `refreshAccessToken` so that all `this.token` setting lives together

Reply:
that's reasonable, I can make that change.